src/data/pages/marketing/index.ts

```diff
@@ -4,84 +4,84 @@ import type { BaseFeature } from '/@src/types'
 export const heroAvatars: HeroAvatar[] = [
   {
     id: 0,
-    picture: 'data:image/gif;base64,replace_with_your_image',
+    picture: '/public/platforms/discord.png',
```

Comment:
Esta es la forma correcta de apuntar a la imagen

```suggestion
    picture: '/platforms/discord.png',
```